python/cudf/cudf/pandas/_wrappers/pandas.py

```diff
@@ -1811,7 +1875,19 @@ def initial_setup():
     cudf.Index.__init__ = wrap_init(_original_Index_init)
     cudf.DataFrame.__init__ = DataFrame_init_
     cudf.core.index.IndexMeta.__call__ = wrap_call(_original_IndexMeta_call)
-
+    cudf.from_pandas = wrap_from_pandas(_original_from_pandas)
+    cudf.DataFrame.from_pandas = wrap_from_pandas_dataframe(
+        _original_DataFrame_from_pandas
+    )
+    cudf.Series.from_pandas = wrap_from_pandas_series(
+        _original_Series_from_pandas
+    )
+    cudf.BaseIndex.from_pandas = wrap_from_pandas_index(
+        _original_Index_from_pandas
+    )
+    cudf.MultiIndex.from_pandas = wrap_from_pandas_multiindex(
+        _original_MultiIndex_from_pandas
+    )
```

Comment:
And then use `functools.partial`
```suggestion
    cudf.from_pandas = partial(wrap_from_pandas, type=None)
    cudf.DataFrame.from_pandas = partial(wrap_from_pandas, type_=cudf.DataFrame)
    ...
```